app/models/reducer.rb

```diff
@@ -40,19 +45,67 @@ def self.of_type(type)
 
   NoData = Class.new
 
-  def process(extracts)
+  # def process(extracts, reductions=nil)
```

Comment:
Any reason this commented out line is here?

Reply:
it shouldn't have been!